docs/sql/streaming.md

```diff
@@ -94,11 +94,57 @@ A table or view can have any number of columns annotated with
 lateness.  An inserted row is considered "too late" if any of its
 annotated columns is too late.
 
+## `emit_final` views
+
+:::warning
+
+The `emit_final` feature is still experimental, and it may be removed
+or substantially modified in the future.
+
+:::
+
+A view can have the property `emit_final` attached, as in the
+following example:
+
+```sql
+CREATE VIEW V WITH (
+  'emit_final' = 'ts'
+) AS SELECT ... as ts, .... FROM ...
+```
+
+This property is modeled after the Kafka [emit on
+close](https://kafka.apache.org/33/javadoc/org/apache/kafka/streams/kstream/EmitStrategy.html#onWindowClose())
+property.
+
+The `emit_final` property must specify either a column name that
+exists in the view, or a column number, where 0 is the leftmost view
+column.
+
```

Comment:
```suggestion
The effect of the `emit_final` annotation is to cause the views to
emit only the rows that have a value in the specified column that is
before the current [waterline](), i.e., these rows will never be updated.
If the compiler cannot infer a waterline for the specified column of
the view, it will emit an error.
```